aiida_vasp/utils/workchains.py

```diff
@@ -120,3 +120,26 @@ def compose_exit_code(status, message):
     """Compose an ExitCode instance based on a status and message."""
     exit_code = ExitCode(status=status, message=message)
     return exit_code
+
+
+def site_magnetization_to_magmom(site_dict):
+    """
+    Convert site mangetization to MAGMOM used for restart
+    NOTE: Only tested for colinear cases
```

Comment:
Should we raise something for non-collinear? We now have a flag for this so that should be rather easy.

Reply:
I have not had any chance to test with those. Also, I am wondering if there are other keywords other than `sphere`. @JPchico  and idea how it should be handled?

Reply:
Yes, I also wonder about the `LNONCOLLINEAR` flag. Have not checked yet.

Reply:
At least I think we at the workchain or calculation point we _know_ we are sort of in noncollinear mode. We anyway have to adjust the executable so we somehow need to modify and know about the code.